README.md

```diff
@@ -25,3 +31,23 @@ export DOTNET_ENVIRONMENT=Development
 ```bash
 cd src && dotnet build
 ```
+
+## Contributing
+
+For guidance on making a contribution to MONAI Deploy App SDK, see the [contributing guidelines](https://github.com/Project-MONAI/monai-deploy/blob/main/CONTRIBUTING.md).
```

Comment:
@dbericat should say workflow manager instead of app SDK

Reply:
Yes, sorry. It actually has to point to MONAI Deploy parent repo